docs/Assets.md

```diff
@@ -1,26 +1,24 @@
+UI5 Web Components need non-code assets for their enterprise-ready features (i18n texts, themes, etc.). These assets need to be known at runtime, but the actual bundle is created by applications. The application has to "link" the assets from their node_modules location to their runtime location and make them known to the UI5 Web Components framework.
 
-The easiest way to enable applications for such "linking" is to express the dependency from the code of the web components to the actual asset files in terms of ES6 imports. In our case, the assets are in `.json` format, so using ES6 imports for the `.json` files is a way to "show" applicaitons what assets are needed.
+The easiest way to enable applications for such "linking" is to express the code dependency of the Web Components to the actual asset files in terms of ES6 imports. In our case, the assets are in `.json` format, so using ES6 imports for the `.json` files is a way to "show" applications what assets are needed.
 
-Since JSON module imports are not standard browser functionality yet (spec work is in progress), a build tool is necessary and the assets imports are not imported automatically from the components in order to make them work without a build tool for rapid prototyping.
+Since JSON module imports are not a standard browser functionality yet (spec work is in progress), you need a build tool. The assets imports are not imported automatically from the components in order to work without a build tool for rapid prototyping.
 
-The assets are still required for productive usage so a separate import is necessary as well.
+You need a separate import as well because the assets are still required for productive usage.
 
-Currently our npm packages follow the scheme:
+Currently, our npm packages follow this scheme:
 
 `@ui5/<PACKAGE_NAME>/dist/generated/assets/*`
 (for the assets themselves)
 
 `@ui5/<PACKAGE_NAME>/dist/Assets.js`
 (for the module that provides the assets)
 
-All JS build tools support importing JSON modules (via plugins like `@rollup/plugin-json` or by default like webpack), by inlining them in a JS module that exports the acutual content. In order to avoid inlinding all 40+ languages and ending up with a huge bundle, the `Assets.js` file exposes the assets with dynamic imports. This way, application build tools can do the following two things:
-- include the necessary JSON data (inlined in JS) in the build output folder
-- leave dynamic imports to only load the necessary one at runtime (since the ui5 webcomponents framework will see the runtime location inside the dynamic import).
+All JS build tools support importing JSON modules (via plugins like `@rollup/plugin-json` or by default like webpack), by inlining them in a JS module that exports the actual content. To avoid the inlinding of all 40+ languages and ending up with a huge bundle, the `Assets.js` file exposes the assets with dynamic imports. This way, application build tools can do the following two things:
```

Comment:
To avoid the **inlining** of all...

(A typo)

Reply:
Done